sdk/arc-scvmm/Azure.ResourceManager.Arc.ScVmm/src/autorest.md

```diff
@@ -0,0 +1,56 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: scvmm
+namespace: Azure.ResourceManager.scvmm
+require: https://github.com/Azure/azure-rest-api-specs/blob/ba936cf8f3b4720dc025837281241fdc903f7e4d/specification/scvmm/resource-manager/readme.md
+tag: package-2020-06-05-preview
+output-folder: Generated/
+clear-output-folder: true
+skip-csproj: true
+modelerfour:
+  flatten-payloads: false
+rename-rules:
+  CPU: Cpu
+  CPUs: Cpus
+  Os: OS
+  Ip: IP
+  Ips: IPs
+  ID: Id
+  IDs: Ids
+  VM: Vm
+  VMs: Vms
+  VMScaleSet: VmScaleSet
+  DNS: Dns
+  VPN: Vpn
+  NAT: Nat
+  WAN: Wan
+  Ipv4: IPv4
+  Ipv6: IPv6
+  Ipsec: IPsec
+  SSO: Sso
+  URI: Uri
+directive:
+  - from: scvmm.json
+    where: $.definitions
+    transform: >
+      $.VirtualMachineDeleteCheckpoint.properties["description"] = {
```

Comment:
Temporay add dummy property to workaround the codegen issue, otherwise these 2 models will disapper